cmd/editorconfig-checker/main.go

```diff
@@ -2,56 +2,69 @@
 package main
 
 import (
+	"errors"
 	"flag"
 	"fmt"
+	"io/fs"
 	"os"
 
 	"github.com/editorconfig-checker/editorconfig-checker/v3/pkg/config"
 	"github.com/editorconfig-checker/editorconfig-checker/v3/pkg/error"
 	"github.com/editorconfig-checker/editorconfig-checker/v3/pkg/files"
-	"github.com/editorconfig-checker/editorconfig-checker/v3/pkg/logger"
 	"github.com/editorconfig-checker/editorconfig-checker/v3/pkg/outputformat"
 	"github.com/editorconfig-checker/editorconfig-checker/v3/pkg/utils"
 	"github.com/editorconfig-checker/editorconfig-checker/v3/pkg/validation"
 )
 
 // version is used vor the help
```

Comment:
Not related to this PR, but now that I see the typo, and that the PR is related to refactoring and code clean up anyway, might be worth to fix it:

```suggestion
// version is used for the help
```

Reply:
lol.. I know - once one sees it, one cannot unsee it.. I fixed it, and added a little to the explanation on what that variable is for.